app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -57,6 +57,92 @@ def hash_for_pdf
         "undefined_9": get_line_7_exemption_count,
         "x  1000_2": get_line_7_exemption_count * 1000,
       }
+
+      dependents = get_dependents
```

Comment:
[dust] rubocop is underlining some of the funcs in this file with "Naming/AccessorMethodName: Do not prefix reader method names with `get_`." IMO this is fine, but we could decide to change it for consistency, or turn that rubocop rule off.